src/Compilers/Core/Portable/Operations/OperationExtensions.cs

```diff
@@ -109,10 +109,31 @@ public static ImmutableArray<ILocalSymbol> GetDeclaredVariables(this IVariableDe
             var arrayBuilder = ArrayBuilder<ILocalSymbol>.GetInstance();
             foreach (IVariableDeclaration group in declarationStatement.Declarations)
             {
-                foreach (ILocalSymbol symbol in group.Variables)
-                {
-                    arrayBuilder.Add(symbol);
-                }
+                arrayBuilder.AddRange(group.GetDeclaredVariables());
+            }
+
+            return arrayBuilder.ToImmutableAndFree();
+        }
+
+        public static ImmutableArray<ILocalSymbol> GetDeclaredVariables(this IVariableDeclaration declaration)
```

Comment:
>GetDeclaredVariables [](start = 51, length = 20)

Consider adding a private overload that takes a builder and simply adds symbols into it. The method above and this one should delegate the work to it.